server/server.js

```diff
@@ -118,3 +118,12 @@ app.get('/server/status', (req, res) => {
   const outputJson = { status: 'True' };
   res.status(200).send(outputJson);
 });
+
+app.get('*', (req, res) => {
+  res.render(path.join(__dirname, '..', 'client', 'page_404'), {
+    header: 'Oops...',
+    // eslint-disable-next-line object-shorthand
```

Comment:
Don't think we should add anymore eslint-disable lines unless we have a specific reason.